numpy/core/src/umath/ufunc_object.c

```diff
@@ -5572,6 +5603,8 @@ ufunc_get_identity(PyUFuncObject *ufunc)
         return PyInt_FromLong(1);
     case PyUFunc_Zero:
         return PyInt_FromLong(0);
+    case PyUFunc_MinusOne:
```

Comment:
`assign_reduce_identity_minusone()` is used to produce a number later passed to `PyLong_AsLongLong` in `PyArray_FillWithScalars` and  I want to be sure that the maximum number of bits are produced. Might could use something smaller, maybe not, but why gamble?